scripts/ai/tasks/CpAITaskFieldWork.lua

```diff
@@ -4,13 +4,51 @@ CpAITaskFieldWork = CpObject(CpAITask)
 
 function CpAITaskFieldWork:reset()
 	self.startPosition = nil
+	self.waitingForRefuelActive = false
 	CpAITask.reset(self)
 end
 
 function CpAITaskFieldWork:setStartPosition(startPosition)
 	self.startPosition = startPosition
 end
 
+function CpAITaskFieldWork:setWaitingForRefuelActive()
```

Comment:
Refuel can be confusing, it sounds more like filling the tractor with diesel. Can we use refill everywhere?